File: glingler-user-web-app/src/app/dto/profile-d-t-o.ts
export class ProfileDTO {
  constructor(
    public profileId,
    public firstName,
    public lastName,
    public bio,
    public sex,
    public age,
    public birthday,
    public imageUrl,
    public createdTime,
    public userId,
    public ageRangeStart,
    public ageRangeEnd,
    public location,
    public lookingFor,
    public status,
    public matchCount,
  ) {
  }
}

File: glingler-api-manager/src/main/java/lk/ijse/glingler/dto/ProfileDTO.java
package lk.ijse.glingler.dto;

import lombok.Data;

@Data
public class ProfileDTO {
    private int profileId;
    private String firstName;
    private String lastName;
    private String bio;
    private String sex;
    private int age;
    private String birthDay;
    private String imageUrl;

    private String lookingFor;
    private String location;
    private Integer ageRangeStart;
    private Integer ageRangeEnd;

    private int matchedCount;
    private int matchedIdWithUser;
}
